### ISSUE TITLE
CUDA 11 support?

### ISSUE THREAD

**Issue by oscarbg** [NVIDIA/jitify#69]
just asking..

**Comment by ptheywood** [comment]
This appears to be due to the expected change in NVRTC no longer having `/usr/include` on the header search path.

> In previous CUDA toolkits, NVRTC on Linux incorrectly added "/usr/include" to the default header file search path. This issue has been fixed; NVRTC in CUDA 11.0 and later will not implicitly add '/usr/include' to the header file search path.
>
> If some included files are present inside /usr/include, the NVRTC nvrtcCompileProgram() API call must now be explicitly passed the "/usr/include" path with the "-I" flag.
> [source](https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html#cuda-compiler-new-features)

The jitify compiler error we see is:

```
stdlib.h(3): catastrophic error: could not open source file "stddef.h" (no directories in search list)
```

**Comment by benbarsdell** [comment]
I think this is actually a different issue, and it should have been fixed with https://github.com/NVIDIA/jitify/pull/52/commits/40df041ce44c5a2dbeb8e2dacc5acad94066ee22. Do you have that commit?

**Comment by ptheywood** [comment]
> I think this is actually a different issue, and it should have been fixed with [40df041](https://github.com/NVIDIA/jitify/commit/40df041ce44c5a2dbeb8e2dacc5acad94066ee22). Do you have that commit?

Our jitify version seemed to be a little out of date, now I've updated it CUDA 11 is working fine for our (flamegpu) use.

**Comment by oscarbg** [comment]
I will close as seems it's ready already..
